projects/igniteui-angular/src/lib/select/select.component.ts

```diff
@@ -363,10 +363,12 @@ export class IgxSelectComponent extends IgxDropDownComponent implements IgxSelec
     public selectItem(newSelection: IgxDropDownItemBaseDirective, event?) {
```

Comment:
this method should be called `selectItemAndCloseDropDownIfNeeded` 😢 
We may consider refactor this and call `toggleClose` after call `selectItem`